index.html

```diff
@@ -4975,6 +4975,637 @@ <h2 id="AudioWorkerParamDescriptor">
+                <dt>
+                  DOMString name
+                </dt>
+                <dd>
+                  A string key that represents a class definition to be
+                  registered.
+                </dd>
+                <dt>
+                  Function classConstructor
+                </dt>
+                <dd>
+                  A class definition extended from
+                  <a>AudioWorkletProcessor</a>.
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+        </section>
+        <section>
+          <h2 id="AudioWorkletProcessor">
+            The <dfn>AudioWorkletProcessor</dfn> interface
+          </h2>
+          <p>
+            This interface represents an audio processing module that runs on
+            the audio <a>rendering thread</a>. It is defined in an
+            <a><code>AudioWorkletGlobalScope</code></a> and the definition of
+            the class manifests the actual audio processing mechanism of a
+            custom audio node. <a><code>AudioWorkletProcessor</code></a> can
+            only be instantiated by the construction of an
+            <a><code>AudioWorkletNode</code></a> instance.
+          </p>
+          <dl title="interface AudioWorkletProcessor" class="idl">
+            <dt>
+              readonly attribute static sequence&lt;AudioParamDescriptor&gt;
+            </dt>
+            <dd>
+              <p>
+                This array contains <a>AudioParamDescriptor</a>s, which are
+                descriptors for each controllable parameter on an
+                <a>AudioNode</a>. This static property is referenced in the
+                construction of an <a>AudioWorkletNode</a> to create instances
+                of an <a>AudioParam</a> in the node.
+              </p>
+            </dd>
+            <dt>
+              readonly attribute AudioContextInfo contextInfo
+            </dt>
+            <dd>
+              <p>
+                This getter returns an <a><code>AudioContextInfo</code></a>
+                object from the associated <a>AudioContext</a>.
+              </p>
+            </dd>
+            <dt>
+              attribute EventHandler onmessage
+            </dt>
+            <dd>
+              <p>
+                The <code>onmessage</code> handler is called whenever the
+                <a><code>AudioWorkletNode</code></a> posts a message back to
+                the corresponding processor object.
+              </p>
+            </dd>
+            <dt>
+              void process()
+            </dt>
+            <dd>
+              <p>
+                This method gets executed isochronously by the audio
+                <a>rendering thread</a>.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  Float32Array[][] inputs
+                </dt>
+                <dd>
+                  An Array of Arrays of <code>Float32Array</code>s. The top-
```

Comment:
I guess you still need text to note that the valid values for `m` can change dynamically (due to connections to input `n`).  The valid values of `n` are fixed at construction since we can't dynamically change the number of inputs to a node.